app/editor/src/features/admin/users/UserFormIndirectUser.tsx

```diff
@@ -138,6 +141,59 @@ export const UserFormIndirectUser: React.FC<IUserFormProps> = (props) => {
                 }}
               />
             </Section>
+            <Section>
```

Comment:
Add this to the Indirect User form so that sources and media types can be linked to the Indirect user.